### PR TITLE
fix hyperlinks ergoCub1.1 & ergoCub1.2 - wiring folder

### PR DESCRIPTION
**Pull request by AntonioAzocar**
I've updated links under ergoCub1_wiring and I separated the ergoCub1.1 (SN001) and ergoCub1.2 (SN002) pdfs

cc @pattacini

### CODE REVIEW

**Review by pattacini — APPROVED** [review]
That's great.
Thanks!

File: docs/ergoCub1_wiring/ergoCub1_2.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## ergoCub 1.2 Wiring 
+The system architecture of ergoCub1.2 is depicted in the following image:
+
+<center>
+  <img src ="../img/ergoCub1.1_architecture.png" width=1000>      
+</center>
```
Comment (pattacini) [inline docs/ergoCub1_wiring/ergoCub1_2.md:6]: Hi @AntonioAzocar 

We're still pointing here to the ergoCub 1.1 architecture.
Is this ok?

Comment (AntonioAzocar) [reply docs/ergoCub1_wiring/ergoCub1_2.md:6]: No, that's wrong... now I modify the architecture with another PR

Comment (pattacini) [reply docs/ergoCub1_wiring/ergoCub1_2.md:6]: Related PR:
- #347

**Pull request merged**